test/check-login

```diff
@@ -77,11 +62,15 @@ class TestLogin(MachineCase):
 
         # Try to login as user with correct password
         login ("user", "abcdefg")
-        b.wait_text("#login-error-message", "Permission denied")
+        if 'atomic' in m.os:
+            b.wait_in_text("body", "connect or authenticate: terminated")
```

Comment:
This probably relates to using ssh under the hood. I agree: a proper error message would be good here.